source/technical/reservations.rst

```diff
@@ -17,6 +17,33 @@ Service for OpenStack.
 Three types of resources can be reserved: physical bare metal hosts, network
 segments (VLANs), and floating IPs.
 
+.. attention::
+   **A note on lease stacking**
```

Comment:
I think this text is too long for a note. Some of it should be cut or at least moved to the lease stacking section.